lib/shared/bucketing-assembly-script/assembly/types/dvcUser.ts

```diff
@@ -26,6 +26,7 @@ export class DVCUser extends JSON.Obj implements DVCUserInterface {
     appVersion: string | null
     appBuild: f64
     customData: JSON.Obj | null
+    optIns: JSON.Obj | null
```

Comment:
this isn't something that can be passed in is it? If not it shouldn't be on DVCUser

Reply:
yeah but that would require an sdk update for all the sdks that use this. the idea was that the optIn data is stored in edgeDB as user.optIns, so all the places where we combine edgeDB data with user data would automatically handle it by adding it to the DVCUser object that gets sent for bucketing, without needing an update